src/app/components/attestation/attestation-create.jsx

```diff
@@ -231,18 +231,20 @@ function ChplAttestationCreate(props) {
                   </Typography>
                 </CardContent>
               </Card>
-              <Typography className={classes.forAssistanceContainer} variant="body2">
+              <Typography gutterBottom className={classes.forAssistanceContainer} variant="body1">
                 For assistance with the Attestations submissions process, please visit the
                 {' '}
                 <a href="https://inquiry.healthit.gov/support/plugins/servlet/loginfreeRedirMain?portalid=2&request=51">Health IT Feedback and Inquiry Portal</a>
                 {' '}
                 to submit a ticket as applicable.
               </Typography>
+              <Typography variant='body2'>
```

Comment:
This gives error messages in the console, as `Typography` compiles down to a `<p>` tag, and `<ul>` tags can't be nested inside a paragraph